src/evaluate.ts

```diff
@@ -101,15 +101,15 @@ async function evaluateCodeUpdatingUI(
       line: line + 1,
       column: column + 1,
       stdout: (m) => {
-        outputWindow.append(normalizeNewLines(m));
```

Comment:
I'm guessing the normalizing of newlines is not actually needed? I don't know the history behind it.